jplag/src/main/java/de/jplag/SubmissionSetBuilder.java

```diff
@@ -134,13 +135,15 @@ private Optional<Submission> loadBaseCode(Set<File> submissionDirectories, Set<F
                 }
 
                 // There is one root directory, and the submissionDirectories variable has been checked to be non-empty.
-                // That set thus contains the the one and only root directory.
+                // That set thus contains the one and only root directory.
                 File rootDirectory = submissionDirectories.iterator().next();
 
                 // Single root-directory, try the legacy way of specifying basecode.
                 baseCode = loadBaseCodeViaName(baseCodeName, rootDirectory, foundSubmissions);
             }
+            // TODO DF: Here the method assumes that baseCode can be null. later, this is not assumed anymore
```

Comment:
Ah, this might be a bug introduced during the multi-root PRs. Currently, we do not have an issue for this.